tendrl/alerting/handlers/cluster/brick_utilization_handler.py

```diff
@@ -10,6 +10,48 @@ class BrickUtilizationHandler(AlertHandler):
     def __init__(self):
         AlertHandler.__init__(self)
 
+    def format_alert_message(self):
```

Comment:
@shtripat no the way plugin_instance is used is varying and that is the major factor influencing the message formation..

The reason this is happening is:
1. Stats from collectd read plugins(the stats collection plugins in our case) are passed to collectd write plugins(the write_graphite plugin in our case) by using collectd#values#dispatch
now the collectd#Values does not allow us to pass a lot of meta data which makes it difficult to later figure out the hierarchy of a particular resource for which a utilization alert is detected by collectd. The only where such info fits in collecd scheme of things is collectd#Values#plugin_instance.
2. The point 1 above makes it necessary to have custom format_message for each plugin